rc/tools/doc.asciidoc

```diff
@@ -2,33 +2,35 @@
 
 This is Kakoune's online documentation system.
 
-To see what documentation topics are available, type `:doc` and look at the
-completion menu. To view the a particular documentation topic, type the topic
-name or choose it from the completion menu and hit Enter.
+To see what documentation topics are available,
+type `:doc` and look at the completion menu.
+To view a particular topic,
+type its name or select it from the completion menu.
+Then hit Enter.
```

Comment:
Ok there are two changes in this paragraph
1. convert to one-sentence-per-line
2. grammar fixes (which look good)

I think converting to one-sentence-per-line should be a separate PR, and if
we want it, we should probably change everything under doc/ to maintain
a consistent style. I personally don't think it's worth it. Granted,
one-sentence-per-line makes future diffs less noisy but do we actually
have so many doc changes that it warrants making sources less pleasing to read?

Separating style from content changes would also make this easier to review.

Reply:
I'm new to this. Another developer directed me to some article explaining that we should try to break lines across clauses. So that's what I was aiming for, not at sentences. I can start this revision over again form the beginning.

Reply:
Every project has a different style; the style of this project's *.asciidoc files appears to be to hard-wrap lines at 80 columns (using `|fmt -w 80` for example).

Reply:
> Docs should be easy to read unrendered as well, so I'd like to get back to nicely wrapped paragraphs, can you fix that please ?

I've wrapped all the lines constituting paragraphs at column 80.

How's that?